extensions/jython/src/com/google/refine/jython/JythonEvaluable.java

```diff
@@ -69,7 +70,11 @@ static public LanguageSpecificParser createParser() {
 
             @Override
             public Evaluable parse(String source, String languagePrefix) throws ParsingException {
-                return new JythonEvaluable(source, languagePrefix);
+                try {
+                    return new JythonEvaluable(source, languagePrefix);
+                } catch (PySyntaxError e) {
+                    throw new ParsingException("Syntax error");
```

Comment:
```suggestion
                    throw new ParsingException("Syntax error", e);
```

Thanks for tackling this! I know this has already been merged, but just wanted to note a suggestion to return original exception wrapped by ours, because it has useful information like the line and column number, the more specific Python parser message, etc which will be useful to the user in debugging their error.